examples/storybooks/sandbox-utils.js

```diff
@@ -0,0 +1,74 @@
+import { getParameters } from 'codesandbox/lib/api/define';
+
+const GIT_URL =
+  'https://api.github.com/repos/fritz-c/react-sortable-tree/contents/';
+
+export const SANDBOX_URL = 'https://codesandbox.io/api/v1/sandboxes/define';
+
+// full url for github api call
+const getURL = filename => `${GIT_URL}/examples/storybooks/${filename}`;
+
+// strip ../../src from the src
+const strip = code => code.replace('../../src', 'react-sortable-tree');
+
+const index = `
+import React from 'react';
+import { render } from 'react-dom';
+import App from './example';
+
+render(<App />, document.getElementById('root'));
```

Comment:
Instead of using this as the `index.js` file, could you prepend the `import { render } from 'react-dom';` to the example, append `render(<App />, document.getElementById('root'));` to the example, and use _that_ as `index.js`? I would prefer that just so the user is faced with the example code as soon as they open codesandbox. As it is now, they have to learn the UI and figure out which file to select to play with the code.

Reply:
Yes. That would make it easier for the user!